src/dso_api/dynamic_api/serializers.py

```diff
@@ -662,17 +670,46 @@ def _links_serializer_factory(model: type[DynamicModel], depth: int) -> type[Dyn
         elif isinstance(model_field, models.ManyToManyField):
             _build_m2m_serializer_field(serializer_part, model, model_field)
         elif isinstance(model_field, (RelatedField, ForeignObjectRel, LooseRelationField)):
-            _build_plain_serializer_field(serializer_part, model_field)
+            _build_serializer_links_field(serializer_part, model_field)
 
     # Generate serializer class
     return serializer_part.construct_class(serializer_name, base_class=DynamicLinksSerializer)
 
 
+def _nontemporal_link_serializer_factory(
+    related_model: type[DynamicModel],
+) -> type[serializers.ModelSerializer]:
+    """Construct a serializer that represents a relationship of which the remote
+    table is not temporal."""
+    if related_model.is_temporal():
+        raise ValueError("Use _temporal_link_serializer_factory instead")
```

Comment:
To ensure PyCharm and other editor can refactor this fearlessly without having to depend on textual similarities, please use:
```python
f"Use {_temporal_link_serializer_factory.__name__} instead"
```